doc/tutorials/introduction/table_of_content_introduction.markdown

```diff
@@ -8,7 +8,7 @@ Additionally you can find very basic sample source code to introduce you to the
 
     _Compatibility:_ \> OpenCV 2.0
 
-    _Author:_ Ana Huamán
+    _Author:_ Ana Huamán and Alessandro de Oliveira Faria
```

Comment:
I think we don't change authors line here, it should be read "Original author". All subsequent contributors can be determined via git/github history.

Reply:
I was planning to write for all linux.

Reply:
Done